docs/.vuepress/client.ts

```diff
@@ -8,6 +8,23 @@ const storageKey = "VUEPRESS_TAB_STORE";
 export default defineClientConfig({
     enhance({app, router, siteData}) {
         // Router configuration
+        router.afterEach(() => {
```

Comment:
It won't work as tabs have keys and there might be different key groups (os, lang) per page. We should not switch all of them.

Reply:
I believe the expected values for `code` in `VUEPRESS_TAB_STORE` will be programming languages like `Java`, `C#`, `TypeScript`, `JavaScript`, `Go`, `Rust`, and `Python`. The logic will only click on the tab with the corresponding label (e.g., `C#`). Therefore, unless there are other types of tabs with conflicting labels, this approach should work as expected.

Below is the list of `@tab` currently used in the codebase:

```
docs/commercial-tools/cli-tool.md
34:@tab Windows
38:@tab Linux and macOS

docs/clients/grpc/observability.md
32:@tab npm
36:@tab yarn
40:@tab pnpm

docs/cloud/automation/terraform.md
46:@tab Linux
48:@tab macOS
50:@tab Windows
310:@tab AWS
312:@tab Azure
314:@tab GCP

docs/clients/grpc/projections.md
29:@tab npm
33:@tab yarn
37:@tab pnpm
48:@tab Maven
57:@tab Gradle

docs/clients/grpc/getting-started.md
26:@tab pip
30:@tab Poetry
41:@tab npm
45:@tab yarn
49:@tab pnpm
62:@tab Maven
70:@tab Gradle

docs/clients/grpc/authentication.md
22:@tab bash
26:@tab PowerShell
56:@tab TypeScript
58:@tab Java
60:@tab C#

docs/clients/tcp/dotnet/21.2/connecting.md
29:@tab JSON format event
31:@tab Plain-text format event
320:@tab Cluster DNS Discovery
326:@tab Gossip Seeds
341:@tab Cluster DNS Discovery
343:@tab Gossip Seeds

docs/cloud/ops/README.md
15:@tab Cloud Console
35:@tab esc
58:@tab Cloud Console
78:@tab esc
103:@tab Cloud Console
113:@tab esc
202:@tab Cloud Console
208:@tab esc
250:@tab Cloud Console
274:@tab esc
315:@tab Cloud Console
325:@tab esc
391:@tab Cloud Console
409:@tab esc

docs/cloud/use/tailscale.md
33:@tab AWS
37:@tab GCP
```